<launch>
    <rosparam file="$(find nachi_conveyor)/config/default_para.yaml" command="load"/>
    <arg name="ns_prefix" default="/robot1" />
    <arg name="arm" default="right_arm" />
    <param name="$(arg ns_prefix)/grasp_time" type="int" value="0" />
    <node pkg="nachi_conveyor" type="conveyor_right_arm_waste" name="right_arm_waste" output="screen">
        <param name="conveyor_id" value="$(arg ns_prefix)/conveyor_id" />
        <param name="conveyor_tracking" value="$(arg ns_prefix)/conveyor_tracking" />
    </node>
</launch>